resource/js/util/PreProcessor/Linker.js

```diff
@@ -1,10 +1,23 @@
+var path = require('path');
 
 export default class Linker {
   process(markdown) {
 
     return markdown
       //.replace(/\s(https?:\/\/[\S]+)/g, ' <a href="$1">$1</a>') // リンク
       .replace(/\s<((\/[^>]+?){2,})>/g, ' <a href="$1">$1</a>') // ページ間リンク: <> でかこまれてて / から始まり、 / が2個以上
+      // Pukiwiki Like Linker
+      // see: https://regex101.com/r/k2dwz3/3
+      .replace(/\[\[(([^(\]\])]+)>)?(.+?)\]\]/g, function(all, group1, group2, group3) {
+        // create url
+        // http(s) から始まっていればそのまま利用、そうでなければ window.location.pathname と連結
+        var url = (group3.match(/^https?:\/\//)) ? group3 : path.join(window.location.pathname, group3);
```

Comment:
> Another thing:
> Is `path.join` really needed on a browser? Can't you connect these string simply with '/'?
> The script size may be reduced by removing `require('path')`

**with `require('path')**
```
-rwxrwxrwx 1 yuki yuki 935859  2月 17 18:06 app.39ed906f17808a5a6b6d.js
```

**without `require('path')**
```
-rwxrwxrwx 1 yuki yuki 933848  2月 17 18:11 app.b0a0426e85fb0f2fb9f4.js
```

I think using `path` is bugless way compared to string operation basically.
But if you value the file size, I'll fix.